### DISCUSSION TITLE
canary route check for default ingress controller are failing for upgrade from 4.10 to 4.10.x

### DISCUSSION THREAD

**Discussion by eselvam** [okd-project/okd#1331] [General]
canary route check for default ingress controller are failing for upgrade from 4.10 to 4.10.x.

We try to upgrade from 4.10.5 to 4.10.28 and it fails with above error.  The ingress reachable from ingress-canary however in the oc get clusteroperator it shows failed. Hence the console and oauth is not working.  Kindly help us here.

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Comment by eselvam** [comment]
Thanks.  I will attach shortly.

**Comment by eselvam** [comment]
after checking further we see below issue is causing cluster operator ingress not to come up.

we could not reach the route in openshift-ingress-canary from openshift-ingress-operator container named ingress-operator.

If we directly hit the openshift-ingress-canary pods ip using curl it is working perfectly.  If the we use route in openshift-ingress-canary, it does not.

could you please help us.

**Comment by eselvam** [comment]
The default ingress controller reports degraded canary router check for default router failing
